chicago/templates/legislation.html

```diff
@@ -100,25 +100,82 @@ <h3 class="no-pad-bottom"><i class='fa fa-fw fa-list-ul'></i> History</h3>
             <thead>
               <tr>
                 <th>Date</th>
-                <th>Action</th>
                 <th>Legislative body</th>
+                <th>Action</th>
+                <th>Votes for</th>
+                <th>Votes against</th>
+                <th>Other votes</th>
               </tr>
             </thead>
             <tbody>
               {% for action in actions %}
                 <tr>
-                  <td class='nowrap text-muted small'>
+                  <td class='nowrap text-muted'>
                     <span datetime='{{action.date }}'>{{action.date}}</span>
                   </td>
-                  <td class="small">
+                  <td>
+                    {{action.organization.link_html|safe}}
```

Comment:
let's use the {{url}} pattern. i'd like to deprecate the link_html stuff.

Reply:
+1